src/mlpack/methods/ann/layer/dropout_impl.hpp

```diff
@@ -38,23 +37,14 @@ void Dropout<InputDataType, OutputDataType>::Forward(
 {
   // The dropout mask will not be multiplied in the deterministic mode
   // (during testing).
-  if (deterministic)
-  {
-    if (!rescale)
-    {
-      output = input;
-    }
-    else
-    {
-      output = input * scale;
-    }
-  }
-  else
-  {
+  if (deterministic) {
+    output = input;
+  } else {
```

Comment:
Can you fix the style here please?

```
if (deterministic)
{
  output = input;
}
else
{
  ...
}
```

Thanks!

Reply:
Forgot that. Done :)